### ISSUE TITLE
Test accessibility with Lighthouse

### ISSUE THREAD

**Issue by Andrey-1992** [Andrey-1992/suspicious-ip-ui#2]
As a developer, I should be able to test and pass the accessibility standards with Lighthouse.
